ssl/internal.h

```diff
@@ -2291,6 +2291,17 @@ struct SSL3_STATE {
   // sending/echoing the post-quantum experiment signal.
   bool pq_experiment_signal_seen : 1;
 
+  // tls_ticket_request_seen is true if the peer sent a ticket request
+  // extension.
+  bool ticket_request_seen : 1;
+
+  // ticket_request_count carries the ticket request count if one was observed.
+  uint8_t ticket_request_count;
+
+  // ticket_request_signal indicates that the client explicitly requested
+  // some number of tickets.
+  bool ticket_request_signal : 1;
```

Comment:
Just from reading this header, it's unclear what the difference between `ticket_request_seen` and `ticket_request_signal` is.